src/app/pages/classes-page/classes-page.component.scss

```diff
@@ -0,0 +1,222 @@
+@use "../../../import" as *;
+
+.circle{
+    height: 30px;
+    width: 30px;
+    border-radius: 30px;
+    background-color:#DDB375;
+    color: black;
+    display:flex;
+    align-items: center;
+    justify-content:center;
+}
+
+.text-description{
+    margin-left:5%;
+    margin-right:5%;
+    margin-top:2%;
+}
+
+.circleClasses{
+    height: 30px;
+    width: 30px;
+    border-radius: 30px;
+    background-color:#DDB375;
+    color: black;
+    display:flex;
+    align-items: center;
+    justify-content:center;
+    margin-top:8%;
+    justify-content: center;
+    align-items: center;
+}
+
+.background-image
+{
+  background-image: url(../../../assets/background_pictures/backgorund_stat.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  background-attachment: fixed;
+}
+
+.content-page
+{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+    padding-bottom:7%;
+}
+
+.selectClass{
+    background-color: #202229;
+    filter: opacity(90%);
+    margin: $globalPaddingTop 0 2rem 0;
+    border-radius: 10px;
+    color: white;
+    margin-right: 20%;
+    margin-left: 8%;
+    text-align: center;
+    padding: 1rem;
+    width: 50vw;
+    padding-bottom: 2rem;
+}
+
+.statScore {
+    margin-left:50px;
+    width:70px;
+}
+
+.statName {
+    width:150px;
+}
+
+.displayValidationButton{
+    display:flex;
+    align-items:center;
+    justify-content:center;
+    margin-top:2%;
+}
+
+.content
+{
+    font-size: 20px;
+}
+
+.selectStats {
+    background-color: #202229;
+            filter: opacity(90%);
+            margin-top: $globalPaddingTop;
+            margin-bottom: 2rem;
+            border-radius: 10px;
+            color: white;
+            margin-left: 8%;
+            text-align: center;
+            padding: 1rem;
+            padding-bottom: 2rem;
+            width: 25vw;
+}
+
+@media screen and (max-width: 768px) {
+    .selectStats {
+        margin-top: 100px;
+        width: 90vw;
+        margin-right: 5vw;
+        margin-left: 5vw;
+    }
+    .content-page{
+        justify-content: center;
+        align-items:center;
+        display:flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+    }
+    .selectClass {
+        width: 90vw;
+        margin-right: 5vw;
+        margin-left: 5vw;
+    }
+}
+
+.modifying
+{
+    width:1000px;
+    height:700px;
+    background-color:#202229;
+    margin-top: 7%;
+    margin-bottom:12%;
+    margin-left:20%;
+    margin-right:20%;
+    border-radius: 10px;
+    color:white;
+    font-size:20px;
+}
+
+.validationButton
+{
+    justify-content: center;
+    align-items:center;
+    text-align: center;
+    background-color: #DDB375;
+    color:black;
```

Comment:
Same as before with the appropriate variable :
Replace color with variable => color : $textColor